src/app/organisms/settings/Settings.jsx

```diff
@@ -87,6 +87,16 @@ function AppearanceSection() {
           )}
           content={<Text variant="b3">Format messages with markdown syntax before sending.</Text>}
         />
+        <SettingTile
+          title="Hide room change events"
+          options={(
+            <Toggle
+              isActive={settings.hideRoomEvents}
+              onToggle={() => { toggleRoomEvents(); updateState({}); }}
+            />
+          )}
+          content={<Text variant="b3">Hide room change messages from room timeline. (Name, Topic, and Avatar)</Text>}
+        />
```

Comment:
Do we really need to setting to hide these event? I think these event are not something that can be used for spam. Also it will always be a good idea to see room topic change in timeline not sure why we really want to hide that.